js/post.js

```diff
@@ -19,7 +19,7 @@
  * Quick edit
  */
 (function( $ ) {
-	$( document ).bind(
+	$( document ).on(
 		'DOMNodeInserted',
 		function( e ) {
```

Comment:
Ok for the bind jQuery shorthand however it seems that `DOMNodeInserted` mutation events is deprecated too
See https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Mutation_events
and https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver